### PR TITLE
fix(wikiwand): syntax highlighting

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
fixes syntax highlighting aswell as some borders too

<details>
<summary>before/after</summary>

<hr>

</details>

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Comment by uncenter** [comment]
This doesn't match our syntax highlighting guidelines, looks like you just made their colors Catppuccinified. Try referencing our other editor highlighting ports.

**Comment by somerand0mcat** [comment]
> This doesn't match our syntax highlighting guidelines, looks like you just made their colors Catppuccinified. Try referencing our other editor highlighting ports.

can i get the link to these syntax highlighting guidelines?

**Comment by somerand0mcat** [comment]
> This doesn't match our syntax highlighting guidelines, looks like you just made their colors Catppuccinified. Try referencing our other editor highlighting ports.

@uncenter i've changed the syntax highlighting based off some userstyles, i've also updated the screenshot

**Comment by uncenter** [comment]
Still doesn't look right, this is what I'm getting with Shiki/VS Code:
CleanShot 2024-08-14 at 16 03 11@2x

**Comment by somerand0mcat** [comment]
does this work?
image

**Comment by somerand0mcat** [comment]
> Looks better but let's just use https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#pygments lol.

i just put in the code for the pygments and the syntax highlighting instantly gets themed lol

**Comment by somerand0mcat** [comment]
done

File: styles/wikiwand/catppuccin.user.css [hunk 1]
```diff
@@ -368,6 +398,9 @@
     a.wl {
       color: @accent-color !important;
     }
+    .wm_content__0bruk .mw-highlight, pre {
+      border-color: @surface2 !important;
+    }
```
Comment (uncenter) [inline styles/wikiwand/catppuccin.user.css:403]:
```suggestion
    pre, code {
      border-color: @surface2 !important;
    }
```

File: styles/wikiwand/catppuccin.user.css [hunk 2]
```diff
@@ -72,6 +101,7 @@
     --color-bg: @base;
     --color-text: @text;
     --color-grey: @subtext0;
+    --color-subtle: @subtext0;
```
Comment (uncenter) [inline styles/wikiwand/catppuccin.user.css:104]: What does this theme?

**Review by uncenter — COMMENTED** [review]
Good job on adding the Pygments syntax highlighting, just these two comments :)

File: styles/wikiwand/catppuccin.user.css (hunk 2 above)
Comment (somerand0mcat) [reply styles/wikiwand/catppuccin.user.css:104]: themes this
image

Comment (uncenter) [reply styles/wikiwand/catppuccin.user.css:104]: Mind sending me the link you found it on? Just want to see how it looks in context!

Comment (somerand0mcat) [reply styles/wikiwand/catppuccin.user.css:104]: sure, https://www.wikiwand.com/en/%22Hello,_World!%22_program

File: styles/wikiwand/catppuccin.user.css [hunk 3]
```diff
@@ -368,6 +398,9 @@
     a.wl {
       color: @accent-color !important;
     }
+    pre, code {
```
Comment (uncenter) [inline styles/wikiwand/catppuccin.user.css:401]:
```suggestion
    pre, code, samp, kbd {
```
From your example above actually.

File: styles/wikiwand/catppuccin.user.css
Comment (uncenter) [inline styles/wikiwand/catppuccin.user.css:1]: Why was this removed?

Comment (somerand0mcat) [reply styles/wikiwand/catppuccin.user.css:1]: i thought we didn't need the variable (i was really tired)

**Review by uncenter — APPROVED** [review]
Great work! Looks wayy better now. And inline code blocks are still unthemed, but that should be fixed for us by https://github.com/catppuccin/python/pull/69.
